### ISSUE TITLE
Debian Packages errors

### ISSUE THREAD

**Issue by oomek** [mxe/mxe#1772]
```
(Reading database ... 387400 files and directories currently installed.)
Preparing to unpack .../mxe-x86-64-w64-mingw32.static-gtk3_3.22.7-20170208_amd64.deb ...
Unpacking mxe-x86-64-w64-mingw32.static-gtk3 (3.22.7-20170208) ...
dpkg: error processing archive /var/cache/apt/archives/mxe-x86-64-w64-mingw32.static-gtk3_3.22.7-20170208_amd64.deb (--unpack):
 trying to overwrite '/usr/lib/mxe/usr/x86_64-w64-mingw32.static/lib/gailutil.def', which is also in package mxe-x86-64-w64-mingw32.static-gtk2 2.24.29-20170208
Preparing to unpack .../mxe-x86-64-w64-mingw32.static-hdf-eos5_1.15-20170208_amd64.deb ...
Unpacking mxe-x86-64-w64-mingw32.static-hdf-eos5 (1.15-20170208) ...
dpkg: error processing archive /var/cache/apt/archives/mxe-x86-64-w64-mingw32.static-hdf-eos5_1.15-20170208_amd64.deb (--unpack):
 trying to overwrite '/usr/lib/mxe/usr/x86_64-w64-mingw32.static/include/bcea.h', which is also in package mxe-x86-64-w64-mingw32.static-hdf-eos2 19v1.00-20170208
dpkg-deb: error: subprocess paste was killed by signal (Broken pipe)
Preparing to unpack .../mxe-x86-64-w64-mingw32.static-id3lib_3.8.3-20170208_amd64.deb ...
Unpacking mxe-x86-64-w64-mingw32.static-id3lib (3.8.3-20170208) ...
dpkg: error processing archive /var/cache/apt/archives/mxe-x86-64-w64-mingw32.static-id3lib_3.8.3-20170208_amd64.deb (--unpack):
 trying to overwrite '/usr/lib/mxe/usr/x86_64-w64-mingw32.static/lib/libz.a', which is also in package mxe-x86-64-w64-mingw32.static-zlib 1.2.11-20170208
Preparing to unpack .../mxe-x86-64-w64-mingw32.static-pdcurses_3.4-20170208_amd64.deb ...
Unpacking mxe-x86-64-w64-mingw32.static-pdcurses (3.4-20170208) ...
dpkg: error processing archive /var/cache/apt/archives/mxe-x86-64-w64-mingw32.static-pdcurses_3.4-20170208_amd64.deb (--unpack):
 trying to overwrite '/usr/lib/mxe/usr/x86_64-w64-mingw32.static/lib/libpanel.a', which is also in package mxe-x86-64-w64-mingw32.static-ncurses 0.e14300b-20170208
No apport report written because MaxReports is reached already
                                                              Errors were encountered while processing:
 /var/cache/apt/archives/mxe-x86-64-w64-mingw32.static-gtk3_3.22.7-20170208_amd64.deb
 /var/cache/apt/archives/mxe-x86-64-w64-mingw32.static-hdf-eos5_1.15-20170208_amd64.deb
 /var/cache/apt/archives/mxe-x86-64-w64-mingw32.static-id3lib_3.8.3-20170208_amd64.deb
 /var/cache/apt/archives/mxe-x86-64-w64-mingw32.static-pdcurses_3.4-20170208_amd64.deb
E: Sub-process /usr/bin/dpkg returned an error code (1)

```
Is it something I should be worried about?

**Comment by starius** [comment]
gtk2 and gtk3 packages do have both this file: usr/x86_64-w64-mingw32.static/lib/gailutil.def

`build-pkg` mitigates such problems for packages depending on each other, but gtk2 and gtk3 do not depend on each other. I think such aspect of `build-pkg` is a bug. We should fix it (don't know how yet) and add a check that no Debian packages files with the same name.

**Comment by tonytheodore** [comment]
> I think such aspect of build-pkg is a bug

Apart from #1844, there's not a lot `build-pkg` can do - these are bugs in the individual packages or a more general bug of not having support for side-by-side or alternate installs.

 - `gailutil.def` isn't really needed (and certainly not for static builds) so can be deleted in build cleanup
 - `id3lib` seems to use it's own internal version of `zlib`
 - `libpanel` is tricky as these are alternate implementations

**Comment by tonytheodore** [comment]
These should all be fixed in #1844, review after next package build.
